README.md

```diff
@@ -134,15 +134,17 @@ That process is a little tricky, but here's one way to do it: assuming this code
 
     qvm-run --pass-io sd-dev 'tar -c -C /home/user/projects securedrop-workstation' | tar xvf -
 
+(Be sure to include the space after the parent directory.)
```

Comment:
Might be worth clarifying what you mean by `parent directory`, or providing the example path explicitly - this is the first time it's mentioned.